Lunch/ViewController.swift

```diff
@@ -24,5 +31,37 @@ class ViewController: UIViewController {
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
         
     }
+
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.items.count
+    }
+
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell:UITableViewCell = self.lunchTableView.dequeueReusableCellWithIdentifier("LunchItemCell") as! UITableViewCell
+
+        cell.textLabel?.text = self.items[indexPath.row]
+
+        return cell
+    }
+
+    func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        // TODO: add stuff here man
+    }
+
+    // MARK: UITextFieldDelegate
+
+    func textFieldShouldReturn(textField: UITextField) -> Bool {
+        textField.resignFirstResponder()
+        return true
+    }
+
+    func textFieldDidEndEditing(textField: UITextField) {
+        items.append(self.lunchTextField.text)
+        var indexSet: NSIndexSet = NSIndexSet(index: 0)
```

Comment:
You can leave out the `NSIndexSet` on the lefthand side because Swift can infer the type from the right